core/domain/stats_domain.py

```diff
@@ -63,13 +66,119 @@
 MAX_ANSWER_DETAILS_BYTE_SIZE = 10000
 
 
+class SubmittedAnswerDict(TypedDict, total=False):
+    """Dictionary representing the SubmittedAnswer object."""
+
+    answer: str
```

Comment:
Is an answer always a string? I thought we accept answers in various different formats.

Reply:
Done (changed to Any)